modules/page-memory/src/main/java/org/apache/ignite/internal/pagememory/util/PartitionlessLinks.java

```diff
@@ -92,4 +100,45 @@ public static void writeToBuffer(ByteBuffer buffer, long link) {
 
         buffer.putInt(pageIndex(link));
     }
+
+    /**
+     * Reads a partitionless page ID from the memory.
+     *
+     * @param addr Address in memory.
+     * @param off Offset from {@code addr} in bytes.
+     * @param partId Partition ID.
+     * @return Page ID with partition ID.
+     */
+    public static long readPartitionlessPageId(long addr, int off, int partId) {
```

Comment:
What's the difference between this and `readPartitionlessLink`?
There should be no difference. Not only this implementation is buggy, it's also a copy-paste.
Please make a single method and update comment in it. Renaming here would be fine

Reply:
Fix it